include/unifex/config.hpp

```diff
@@ -60,8 +60,7 @@
 #  if __has_include(<coroutine>) && defined(__cpp_lib_coroutine)
 #    define UNIFEX_COROUTINES_HEADER <coroutine>
 #    define UNIFEX_COROUTINES_NAMESPACE std
-#  elif __has_include(<experimental/coroutine>) && \
-        !(defined(_MSC_VER) && defined(__clang__)) // clang-cl.exe
+#  elif __has_include(<experimental/coroutine>)
```

Comment:
Why this change? I don't remember the history here, but we've had a whole bunch of problems with MSVC-specific stuff so I'm hesitant to enable coroutine support where it's presently disabled.
```suggestion
#  elif __has_include(<experimental/coroutine>) && \
        !(defined(_MSC_VER) && defined(__clang__)) // clang-cl.exe
```